src/components/Page/PagePicker.vue

```diff
@@ -5,7 +5,7 @@
 				{{ t('collectives', 'Copy or move page') }}
 			</h2>
 			<span class="crumbs">
-				<div class="crumbs-home">
+				<div v-if="!selectedCollective || !selectedCollective.isPageShare" class="crumbs-home">
 					<NcButton type="tertiary"
 						:aria-label="t('collectives', 'Breadcrumb for list of collectives')"
 						:disabled="!selectedCollective"
```

Comment:
Well, not exactly. The "list of collectives" breadcrumb is displayed if either `!selectedCollective` (no collective selected) or `!selectedCollective.isPageShare` (a collective is selected and this is not a page share). In other words, I only want to display it always, except for page shares. Unfortunately, `!selectedCollective.isPageShare` is not enough here, because `selectedCollective` might be null and in this case I also want to display the button.

The `disabled` condition on the other hand is different: the button should be disabled if no collective is selected.

Reply:
I agree though that the code is not exactly intuitive :zany_face:

Reply:
I think i just confused the `||` for an `&&`